docs/source/troubleshooting.md

```diff
@@ -1,32 +1,37 @@
 # Troubleshooting
 
 When troubleshooting, you may see unexpected behaviors or receive an error
-message. This section provide links for identifying the cause of the
+message. This section provides links for identifying the cause of the
 problem and how to resolve it.
 
 [_Behavior_](#behavior)
 
-- JupyterHub proxy fails to start
```

Comment:
All these exact markdown links are sensitive to character changes in headings, and most [don't actually resolve](https://jupyterhub--4085.org.readthedocs.build/en/4085/troubleshooting.html).

Maybe we should remove all of these? They are redundant with the always-correct autogenerated TOC in the sidebar:

<img width="989" alt="Screen Shot 2022-10-12 at 13 27 28" src="https://user-images.githubusercontent.com/151929/195331198-10534884-3cb2-4c48-a6b8-8d9bde70cfb0.png">

Reply:
@Goodiec, I believe what @minrk is suggesting is get rid of them entirely, including the bullet point lists.
They will still get autogenerated in the sidebar on the right.

<img width="587" alt="Screenshot 2022-10-14 at 13 00 46" src="https://user-images.githubusercontent.com/7579677/195820927-767bd12c-a591-4215-8695-025dc0f48364.png">

Reply:
Got it, I'll update this shortly!